.github/workflows/appimage.yml

```diff
@@ -82,7 +82,7 @@ jobs:
     - name: Build AppImage (PR)
       if: contains(github.event.pull_request.labels.*.name, 'appimage') || contains(github.event.pull_request.labels.*.name, 'appimage-ignore-tests')
       run: |
-        sudo -u testbot bash -lc 'cd viam-cartographer && make BUILD_CHANNEL="pr-${{ github.event.pull_request.number }}" appimage'
+        sudo -u testbot bash -lc 'cd viam-cartographer && make BUILD_CHANNEL="pr-${{ github.event.pull_request.number }}" appimage-custom-build-channel'
```

Comment:
[q] Is the reason the custom PR builds are left in a separate workflow from the rest of the app images builds because it is triggered by pull_request_target and not pushes/workflowdispatch

Reply:
There are probably several ways to organize it, but I thought this was relatively clear. Happy to take suggestions if you think something else would work better!

Reply:
Sounds good, thanks!